src/shared_modules/content_manager/README.md

```diff
@@ -0,0 +1,341 @@
+# Content Manager
+
+The Content Manager is a module that is in charge of obtaining the Wazuh Content from an external repository, as well as maintaining the periodic updates that the manager will obtain.
+
+## Usage
+
+The Content Manager counts with a [test tool](./testtool/main.cpp) that can be used to perform tests, try out different configurations, and to better understand the module.
+
+The input configuration of the Content Manager is described below:
+
+- `topicName`: Topic name, used to represent the actions being executed.
+- `interval`: Interval, in seconds, between each action execution.
+- `ondemand`: If true, the module will be executed on demand.
+- `configData`: Configuration data to create the orchestration of the module.
+  + `contentSource`: Source of the content. Can be any of `api`, `cti-api`, `file`, or `offline`.
+  + `compressionType`: Compression type of the content. Can be any of `gzip`, `zip`, or `xz`.
+  + `versionedContent`: Type of versioned content. Can be any of `false` (content versioning disabled) or `cti-api`.
+  + `deleteDownloadedContent`: If true, the downloaded content will be deleted after being processed.
+  + `url`: URL from where the content will be downloaded or copied.
+  + `outputFolder`: If defined, the content (downloads and uncompressed content) will be downloaded in this folder.
+  + `dataFormat`: Content data format. Examples: `json`, `xml`, `txt`, etc.
```

Comment:
Improve: How many other formats are supported? I would remove the etc all together as it adds no useful information

Reply:
In fact, any `dataFormat` is supported, that's why the "etc.". The `dataFormat` is used to make a replacement in some decompressors for the output files.

So, leaving just "json, xml, txt" doesn't seem right. I could remove all of them, as an alternative. 
What do you think?